proactive-jupyter-kernel/kernel.py

```diff
@@ -91,6 +93,10 @@ def __init__(self, **kwargs):
         self.imports = {}
         self.default_selection_script = None
         self.default_fork_env = None
+        self.multiblock_task_config = False
```

Comment:
Please provide for the main usage of each of those variables, to ensure code readibility and maintanability from non-knowledgeable dev !

Reply:
Yes! I know I should. I will do it. Thanks!